README.md

```diff
@@ -115,75 +96,26 @@ To get a local copy up and running, follow these steps.
 
 In order to run this project you need:
 
-<!--
-Example command:
+- [ ] A Web Browser(Google chrome)
+- [ ] Integrated Development Environment(VSCode)
 
-```sh
- gem install rails
-```
- -->
 
 ### Setup
 
 Clone this repository to your desired folder:
-
-<!--
-Example commands:
-
-```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
-```
---->
+https://github.com/JohnteFresh/It-s-me-again.git
 
 ### Install
 
 Install this project with:
+This project does not need installation. It automatically runs on the web browser.
 
-<!--
-Example command:
-
-```sh
-  cd my-project
-  gem install
-```
---->
-
-### Usage
-
-To run the project, execute the following command:
 
-<!--
-Example command:
-
-```sh
-  rails server
-```
---->
-
-### Run tests
-
-To run tests, run the following command:
-
-<!--
-Example command:
-
-```sh
-  bin/rails test test/models/article_test.rb
-```
---->
-
-### Deployment
-
-You can deploy this project using:
```

Comment:
- [OPTIONAL] Please add back in your `README` file because except FAQ section rest everything is required which makes your README file looks more professional.